.circleci/config.yml

```diff
@@ -440,7 +440,7 @@ workflows:
       - Check Rust dependencies
     triggers:
       - schedule:
-          cron: "0 * * * *"
+          cron: "0 7 * * *"
```

Comment:
I picked 7am at random, seemed polite not to schedule it at midnight like everyone else does.